cxx4opencl/diff2cxx.txt

```diff
@@ -2,11 +2,10 @@
 // Creative Commons Attribution 4.0 International License; see
 // http://creativecommons.org/licenses/by/4.0/
 
-[[restrictions]]
-=== Restrictions
+[[diff2cxx]]
+=== Difference to {cpp}
 
-This section describes various restrictions and limitations for
-language features from OpenCL v2.0 and {cpp}17.
+{cpp} for OpenCL supports the majority of standard {cpp}17 features. However, there are some differences that are documented in this section.
```

Comment:
should be , however.

Reply:
updated!

Also added a bit more explanation about the next section.